### PR TITLE
Update kinship and inbreeding coefficient tests

### PR DESCRIPTION
**Pull request by labkey-martyp**
#### Rationale
Kinship calculations have been updated. Update the genetic tests here and add more thorough testing of kinship and inbreeding results.

#### Related Pull Requests
* https://github.com/LabKey/ehrModules/pull/493

#### Changes
* Add specific pedigree scenarios to the test data
* Verify kinship coefficients and inbreeding coefficents of those scenarios

### CODE REVIEW

File: onprc_ehr/test/src/org/labkey/test/tests/onprc_ehr/ONPRC_EHRTest.java [hunk 1]
```diff
@@ -1258,20 +1276,42 @@ public void testGeneticsPipeline() throws Exception
...
         animalHistoryPage.refreshReport();
         animalHistoryPage.clickCategoryTab("Genetics")
                 .clickReportTab("Kinship");
 
+        log("verify kinship");
         assertTrue(isElementPresent(Locator.linkWithText("CLICK HERE TO LIMIT TO ANIMALS IN SELECTION")));
 
         DataRegionTable kinshipTable = animalHistoryPage.getActiveReportDataRegion();
-        assertEquals("Incorrect number of rows before limiting animal selection", 3, kinshipTable.getDataRowCount());
+        assertEquals("Incorrect number of kinship rows before limiting animal selection", 48, kinshipTable.getDataRowCount()); // TODO: should be more
```
Comment (labkey-jeckels) [inline onprc_ehr/test/src/org/labkey/test/tests/onprc_ehr/ONPRC_EHRTest.java]: Is this TODO relevant?

**Review by labkey-jeckels — CHANGES_REQUESTED** [review]
The update test is failing on a duplicate row problem during the inserts.

**Review by labkey-tchad — COMMENTED** [review]
The test failure is because `createBirthRecords` no longer calls `deleteIfExists` before inserting birth records.
If you make `_hasCreatedBirthRecords` static, `createBirthRecords` should no-op for the second test to call it.
Alternatively, you could have `doSetup` create the birth records instead of making the individual tests do it. I can't imagine it takes terribly long, since its all done via API.

**Review by labkey-jeckels — APPROVED** [review]

File: onprc_ehr/test/src/org/labkey/test/tests/onprc_ehr/ONPRC_EHRTest.java (hunk 1 above)
Comment (labkey-martyp) [reply onprc_ehr/test/src/org/labkey/test/tests/onprc_ehr/ONPRC_EHRTest.java]: Removed

**Pull request merged**